test/request.test.ts

```diff
@@ -447,7 +451,7 @@ x//0u+zd/R/QRUzLOw4N72/Hu+UG6MNt5iDZFCtapRaKt6OvSBwy8w==
   });
 
   it("passes node-fetch options to fetch only", () => {
```

Comment:
Remove `timeout` parameter tests

Reply:
Removed it from the tests in [ea7c2ab](https://github.com/octokit/request.js/pull/580/commits/ea7c2abf255e056eca4aa35a8cb92a06802c794c)

Reply:
These tests don't really seem useful now. Could you remove them

Reply:
Sounds good! Removed in [3cfc7c8](https://github.com/octokit/request.js/pull/580/commits/3cfc7c87c935a82987ff5a3d0227f35998411b1e)